### DISCUSSION TITLE
[Simple] Add support for vanished players on squaremap

### DISCUSSION THREAD

**Discussion by RSLuke** [Zrips/CMI#7731] [Feature Requests]
Their doc :
https://javadoc.io/doc/xyz.jpenilla/squaremap-api/latest/xyz/jpenilla/squaremap/api/PlayerManager.html

**Comment by CloudeLecaw** [comment]
its the job of that map plugin to add Support to cmis vanished people
you may should ask the devs on thieir git...

**Comment by RSLuke** [comment]
Actually it isn't, CMI didn't provide any support to make that possible for squaremap

**Comment by CloudeLecaw** [comment]
> Actually it isn't, CMI didn't provide any support to make that possible for squaremap

thats not true. cmi has an official API that is open source on the git
https://www.zrips.net/cmi/api/

You can download CMI-API from [github.com/Zrips/CMI-API](https://github.com/Zrips/CMI-API)

Dont aspect support for all mapplugins in this world coming from cmi. ;-)
Like i said before its not CMIs jobs to support this.

More than the sqauremap might needs to add support for CMIs vanish.

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions
